tempesta_fw/connection.h

```diff
@@ -54,20 +54,26 @@ enum {
  * Session/Presentation layer (in OSI terms) handling.
```

Comment:
Please add comment that TfwConnection can live longer than underlying TCP connection, for example in connection failovering case. This is important design point which means that TfwConnection could be alive and valid, while sk member in it fully unusable.

Meantime TfwPeer is the property of TfwConnection and it's always the same for the same connection, i.e. TfwConnection has no sense w/o live TfwPeer and can't change its peer.